src/Tasks/dependency-check-build-task/dependency-check-build-task.ts

```diff
@@ -146,8 +149,38 @@ async function run() {
         // Version smoke test
         await tl.tool(depCheckPath).arg('--version').exec();
 
+        if(!hasLocalInstallation) {
```

Comment:
Will remove leftover lock files from a previous run if this is not a locally installed/custom dependency check tool